scripts/fetch_istio_releases.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+RELEASE_DIR=${1}
+istio_version_array=(1.7.1 1.7.3 1.8.3 1.8.5 1.8.6 1.9.3 1.9.5 1.9.6 1.9.8 1.10.4)
+echo ${RELEASE_DIR}
+
+if [ -z "${RELEASE_DIR}" ]; then
+  echo "No directory given"
+  exit 1
+fi
+
+if [ -d "${RELEASE_DIR}" ] ; then
+  cd ${RELEASE_DIR}
+
+  dir=$(pwd)
+
+  for v in "${istio_version_array[@]}"; do
+    curl -sOL https://github.com/istio/istio/releases/download/$v/istio-$v-linux-amd64.tar.gz
+  done 
```

Comment:
I'm slightly worried about the build process getting timed out by github, if it thinks we are spamming it. Can we only grab it for ones that don't exist ? Should be able to check that version path with bash fairly easily.

Reply:
What do you mean grab it for only ones that don't exist? We are now grabbing only a specific set of images - we can maybe change our support for what images will be included in the versions.txt to only be within the version range that is support for upgrades.

Reply:
What I meant was to only download istio versions that don't exist locally already. Though maybe its not possible because if the cache layer gets invalidated then it would start from nothing. Can you at least add a `sleep 1` in the for loop so we don't spam the downloads. 

Also if you do a docker build and then change the versions.txt, does that actually invalidate the docker cache and fetch the versions (including new one) again ?

Reply:
I have gone ahead and added the istio_version_array back into the installer instead of going to version.txt route to avoid all of the issues with cache busting